WORKSPACE

```diff
@@ -131,6 +131,49 @@ rbe_autoconfig(
     use_checked_in_confs = False,
 )
 
+# RBE Autoconfig targets to do integration testing on the automatic toolchain
+# configs release process.
+load(
+    "//configs/tests/dependency-tracking:ubuntu1604_no_updates.bzl",
+    _ubuntu1604_bazel_no_updates = "bazel",
+    _ubuntu1604_configs_version_no_updates = "configs_version",
+    _ubuntu1604_digest_no_updates = "digest",
+    _ubuntu1604_registry_no_updates = "registry",
+    _ubuntu1604_repository_no_updates = "repository",
+)
+
+# Automatic config generation target for RBE Ubuntu 16.04
+rbe_autoconfig(
+    name = "rbe_autoconfig_autogen_ubuntu1604_test_no_updates",
```

Comment:
call this "no_update_test" (or something that is more reflective of the purpose, not the implementation detail)
also, use that (or some other) shorter prefix/suffic for "_ubuntu1604_xxx_no_updates"

Reply:
Done